Cargo.toml

```diff
@@ -60,7 +60,7 @@ criterion = "0.5"
 debug = true
 
 [features]
-default = ["parallel"]
+default = ["parallel", "batching" ]
```

Comment:
I'm not sure if we need a feature flag for this. Generally features are used to gate dependencies and target specific functionality, not actual features :)